website/docs/core-concepts/writing-code/javascript-editor-beta/README.md

```diff
@@ -46,6 +46,39 @@ You can call the functions defined in a JS Object by using the notation `{{ JS_O
 The JS Objects defined are available across APIs, Queries, or other JS Objects defined for **a particular page,** and have a **page-level access** and are **not** accessible **across pages**.
 :::
 
+### JS Object variable as local state
```

Comment:
This shouldn't be under **Calling a JS Object function** It should be under **Create JS Object** under a subheading called **JS Object variables** Under that, show how to define and access the variables.


For local state, this should also be touched in the section where we show reactive programming to update widgets programmatically.